apps/router/src/gateway-ui/components/form/InfoField.tsx

```
@@ -0,0 +1,31 @@
+      borderRadius='md'
+    >
+      <Text fontSize='sm' color='gray.500'>
+        {label}
+      </Text>
+      <Text
+        fontSize='md'
+        fontWeight='medium'
+        textAlign='center'
+        wordBreak='break-all'
+      >
+        <>{value}</>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)